cli/bpmetadata/tfconfig.go

```diff
@@ -279,9 +317,22 @@ func getBlueprintRequirements(rolesConfigPath, servicesConfigPath string) (*Blue
 		return nil, err
 	}
 
+	//parse blueprint provider versions
+	versionsFile, diags := p.ParseHCLFile(versionsConfigPath)
```

Comment:
Could we check if `versionsConfigPath` exists and only proceed if versionsConfigPath exists. Ideally I would like to keep role and services even if versions does not exist.

Reply:
Done